src/app.ts

```diff
@@ -27,11 +28,11 @@ const uri = `mongodb+srv://${process.env.DB_USERNAME}:${process.env.DB_PASSWORD}
 try {
   void mongoose.connect(uri);
   mongoose.connection.once('open', () => {
-    console.log('Database connected ..');
+    logger.info('Database connected ..');
     oddsJob.start();
   });
-} catch (err) {
-  console.log('Database error ..', err);
+} catch (err: any) {
+  logger.info(`Database error .. ${err}`);
```

Comment:
Error messages should be logged at error/warn level.

```suggestion
  logger.error(`Database error .. ${err}`);
```